apiServer/taas/apiserver.go

```diff
@@ -27,5 +27,7 @@ func main() {
 	router.GET("taas/pxversion", utils.GetPxVersion)
 	router.GET("taas/ispxinstalled", utils.IsPxInstalled)
 	router.GET("taas/getpxctloutput", utils.GetPxctlStatusOutput)
+	router.GET("taas/getvmsbyns", utils.GetVMsInNamespaces)
+	router.GET("taas/getvmsbynslabels", utils.GetVMsWithNamespaceLabels)
```

Comment:
Can we please change the API to be "getkubevirtvmsbyns" or an even better name if you can think of - so that just by reading we are aware what the API does.

Reply:
Done